requirements.txt

```diff
@@ -1,2 +1,4 @@
 prefect>=2.7.2
-google-api-python-client>=2.20.0
\ No newline at end of file
+google-api-python-client>=2.20.0
+pandas>=1.0.0
+pyarrow>=11.0.0
```

Comment:
Since this is only for one method, I don't think we should pin as req. But since we do import pandas in tests, we should put in requirements-dev.txt
```suggestion
```

Reply:
Fixed 👍🏻